src/logic/safe/store/actions/fetchSafe.ts

```diff
@@ -70,11 +70,17 @@ export const fetchSafe =
     let safeInfo: Partial<SafeRecordProps> = {}
     let remoteSafeInfo: SafeInfo | null = null
 
-    // if there's no remote info, we keep what's in memory
     try {
       remoteSafeInfo = await getSafeInfo(address)
     } catch (err) {
       err.log()
+
+      // Crash the app when Safe is not found
+      if (err.content === Errors._605) {
+        throw err
```

Comment:
Maybe we should throw `Errors._605` here so the user sees our defined message?

Reply:
The error message returned from backend is apparently `Safe not found, error 1337` so I'll resolve this.